packages/mui-material-next/src/InputBase/InputBase.test.js

```diff
@@ -536,39 +528,14 @@ describe('<InputBase />', () => {
           );
         }).not.toErrorDev();
       });
-
-      it('should not warn when toggling between inputs', () => {
-        // this will ensure that deregistering was called during unmount
-        function ToggleFormInputs() {
-          const [flag, setFlag] = React.useState(true);
-
-          return (
-            <FormControl>
```

Comment:
Was this removed due to the use of `FormControl`? Or the use of `Select`? Or is there another reason?

Reply:
It's because of `Select`, but now I'm wondering if this test should be ported to `FormControl.test` instead

"toggling between inputs (inside a FormControl)" could still be a valid case even the `select` prop on `TextField` is dropped in favor of a SelectField component (https://github.com/mui/material-ui/issues/21782)

What do you think? @DiegoAndai

Reply:
I thought the same; we should move it to `FormControl.test` when we migrate it. It's okay to remove it here, but let's write it down somewhere so we remember to bring it back later.

Reply:
👍 I've added it to https://github.com/mui/material-ui/issues/38411